network-api/networkapi/wagtailpages/pagemodels/customblocks/dear_internet_letter_block.py

```diff
@@ -3,10 +3,13 @@
 
 
 class DearInternetLetterBlock(blocks.StructBlock):
+    author_name = blocks.CharBlock()
+
     author = blocks.RichTextBlock(
```

Comment:
I'd like to rename this to `author_description` but not sure how to do it safely

Reply:
if we've not landed this yet, you should be able to literally just do that, since we won't be losing any preexisting data.